lib/src/rpc/dial.dart

```diff
@@ -234,11 +234,11 @@ Future _logConnectionStats(Stopwatch webrtcDialSW, RTCPeerConnection peerConnect
     if (stat.type == 'candidate-pair' && stat.values['nominated']) {
       // Use 'lastPacketSentTimestamp' on candidate pair to estimate when the
       // pair was nominated.
-      final double lpst = stat.values['lastPacketSentTimestamp'];
+      final double lpst = stat.values['lastPacketSentTimestamp'] ?? 0;
       final DateTime nominatedTime = DateTime.fromMillisecondsSinceEpoch(lpst.toInt());
 
-      final String lcid = stat.values['localCandidateId'];
-      final String rcid = stat.values['remoteCandidateId'];
+      final lcid = stat.values['localCandidateId'];
+      final rcid = stat.values['remoteCandidateId'];
```

Comment:
are you comfortable with these being instantiated as `dynamic`? you'll need to cast them later, I think.

if not, you could do like above:


```suggestion
      final String? lcid = stat.values['localCandidateId'];
      final  String? rcid = stat.values['remoteCandidateId'];
```
or

```suggestion
      final lcid = stat.values['localCandidateId'] ?? ''; // or some default value
      final rcid = stat.values['remoteCandidateId'] ?? '';
```

Reply:
Interesting... they seem to string-interpolate just fine as dynamics; is it preferred for them to be `String?` or `String`?

Reply:
I'm not familiar with Dart, but was there a reason to remove the `String` type?

Reply:
They are not in fact `String`s; they are of type `dynamic` and could be cast to `String?` (nullable `String`).